src/appkit.cs

```diff
@@ -17238,7 +17812,7 @@ partial interface NSTabView {
 		NSTabViewType TabViewType { get; set; }
 
 		[Export ("tabViewItems")]
-		NSTabViewItem [] Items { get; }
+		NSTabViewItem [] Items { get; set; }
```

Comment:
Shouldn't this set have Availability info? what it introduced in 10.14?

Reply:
So the header/docs don't declare 10.14 but I think it's safe to add anyway.